recipes/mosquitto/1.x/conanfile.py

```diff
@@ -0,0 +1,132 @@
+import os
+import glob
+from conans import CMake, ConanFile, tools
+
+required_conan_version = ">=1.29.1"
+
+class MosquittoConan(ConanFile):
+    name = "mosquitto"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/eclipse/mosquitto"
+    topics = ("mqtt", "broker", "libwebsockets", "mosquitto", "eclipse", "iot")
+    license = "EPL-1.0", "BSD-3-Clause" # https://lists.spdx.org/g/Spdx-legal/message/2701
+    description = "Eclipse Mosquitto - An open source MQTT broker"
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
```

Comment:
1.x and 2.x recipes have different options:
- with_tls vs ssl
- with_websockets vs websockets

Better to unify them based on 2.x recipe.

Reply:
2.x should have followed the convention 😞

Reply:
Oh, that's right. 1.x has better naming